lib/deep-equal.test.js

```diff
@@ -365,38 +364,50 @@ describe("deepEqual", function() {
         assert.isFalse(checkDeep);
     });
 
-    it("returns false if object has different symbolic properties", function() {
-        var obj1 = {};
-        var obj2 = {};
-        obj1[symbol] = 42;
-        obj2[symbol] = 43;
-        var checkDeep = samsam.deepEqual(obj1, obj2);
-        assert.isFalse(checkDeep);
-    });
+    context("with symbol properties", function() {
```

Comment:
`describe` is used throughout the file instead of `context.

Reply:
Fixed.